.circleci/config.yml

```diff
@@ -120,3 +145,16 @@ workflows:
     - deploy_uat:
         requires:
         - hold_uat
+    - hold_staging:
```

Comment:
We should probably auto deploy staging and only hold production. What do you think?